src/scripting/lua_widget_methods.cpp

```diff
@@ -293,6 +294,9 @@ static int intf_remove_dialog_item(lua_State* L)
 	} else if(gui2::menu_button* menu_button = dynamic_cast<gui2::menu_button*>(w)) {
 		int realpos = check_index(L, 2, *menu_button, false, pos);
 		menu_button->remove_rows(realpos, number);
+	} else if(gui2::multimenu_button* multimenu_button = dynamic_cast<gui2::multimenu_button*>(w)) {
+		int realpos = check_index(L, 2, *multimenu_button, false, pos);
+		multimenu_button->remove_rows(realpos, number);
```

Comment:
Again here it looks like you can just delete the menu_button and multimenu_button cases, as the options_button case below handles them perfectly.